inst/include/LocalCop/gaussian.hpp

```diff
@@ -14,6 +14,68 @@
 
 namespace LocalCop {
 
+  // TODO: This code is a TMB example, copied here to test how it works. Need to replace it with pgaussian.
```

Comment:
Before doing `pgaussian`, let's do another simple test to make sure we get this right.  The goal is to create a function `integrate_test(x, y, theta)` defined as
```
integrate_test(x, y, theta) = int_0^x z * y * theta dz
```
This function clearly does not need numerical integration to evaluate, since it's just equal to `0.5 * x^2 * y * theta`.  However, it gives us a good test case to see that we are using `gauss_kronrod::Integrate()` correctly.  In particular, one major difference between the `integrate_test()` and the example below is that here we want to differentiate wrt the limits of the integrand as well, which we'll eventually need for `pgaussian`.

Also, please create a file `tests/testthat/test_integrate.R` in which you check that `test_integrate()` and its gradients are correctly implemented in C++/TMB, by comparing to the analytic computations in R.  Please see some of the other files in `tests/testthat` for an example of how to set up such tests.